src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -185,7 +185,7 @@ private function registerCommonConfiguration(ContainerBuilder $container, array
         $container->setParameter('api_platform.collection.exists_parameter_name', $config['collection']['exists_parameter_name']);
         $container->setParameter('api_platform.collection.order', $config['defaults']['order'] ?? $config['collection']['order']);
         $container->setParameter('api_platform.collection.order_parameter_name', $config['collection']['order_parameter_name']);
-        $container->setParameter('api_platform.collection.pagination.enabled', $this->isConfigEnabled($container, $config['defaults']['pagination_enabled'] ?? $config['collection']['pagination']));
+        $container->setParameter('api_platform.collection.pagination.enabled', $config['api_platform']['defaults'] ?? $this->isConfigEnabled($container, $config['collection']['pagination']));
```

Comment:
I'm not sure this is right. If defaults are set, it will be used for the parameter `api_platform.collection.pagination.enabled`.

Reply:
this is correct imo

Reply:
`$config['api_platform']['defaults']` is not a boolean isn't it?

Reply:
it should be `$config['defaults']['pagination_enabled']` indeed, sorry I thought this was the case here